activejob/lib/active_job/logging.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
-require "active_support/tagged_logging"
-require "active_support/logger"
+require "active_support"
```

Comment:
`require "active_support"` already happens in the Active Job entrypoint, so it should not be added here.

https://github.com/rails/rails/blob/800976975253be2912d09a80757ee70a2bb1e984/activejob/lib/active_job.rb#L26

However, you are right that these requires can be removed because they reference an autoloaded constant (`TaggedLogging`) and an already required file (`active_support.rb` requires `active_support/logger`)

Reply:
I agree.  It seems like there are historical patterns around the codebase of Rails which end up getting copied and pasted into other code, even though the paradigm has been dead for a long time.

I've pushed the branch again with the removal of that line as well.